how-to/getting-started-with-r/getting_started_with_r.md

```diff
@@ -0,0 +1,72 @@
+![R logo](https://www.r-project.org/Rlogo.png)
+
+# Getting started with R
+
+R is a free, open-source high-level programming language and software environment used primarily for statistical computing and graphics. 
+
+## R at Healthify:
+R is currently used for web scraping and data processing tasks.
+
+## First steps:
+Start by downloading [R](http://cran.r-project.org/) and [RStudio](http://www.rstudio.com/ide/), the R IDE (optional, but highly recommended).
+
+## Getting help:
+```
+help.start()              # Load HTML help pages into browser
+help(package)             # List help page for "package"
+?package                  # Shortform for "help(package)"
+help.search("keyword")    # Search help pages for "keyword"
+?help                     # For more options
+help(package=base)        # List tasks in package "base"
+demo()                    # List topics for which demos are available
+q()                       # Quit R
+```
+
+## Useful links:
```

Comment:
Yeah - it was at the end but then I added the Unit Test section at the end which I think is OK to have at the end. 
I will rename the header from 'Useful links': to 'Useful links to get started:'